src/pages/workspaces/workspace/ShareWorkspaceModal/Collaborator.ts

```diff
@@ -0,0 +1,156 @@
+import _ from 'lodash/fp';
+import React, { Dispatch, SetStateAction } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { LabeledCheckbox, Link, Select, SelectProps } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import { getPopupRoot } from 'src/components/popup-utils';
+import colors from 'src/libs/colors';
+import { getUser } from 'src/libs/state';
+import * as Utils from 'src/libs/utils';
+import { BaseWorkspace, isAzureWorkspace, WorkspaceAccessLevel } from 'src/libs/workspace-utils';
+import { AccessEntry, WorkspaceAcl } from 'src/pages/workspaces/workspace/WorkspaceAcl';
+
+interface CollaboratorProps {
+  aclItem: AccessEntry; // the item to render
```

Comment:
converting these prop comments to jsdoc comment blocks would better align with most IDE tooltip auto-help for these props as a consumer is coding them up.